plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -252,6 +256,13 @@ public ContainerCreated createContainer(CreateContainerParams createContainerPar
         String[] volumes = createContainerParams.getContainerConfig().getHostConfig().getBinds();
 
         Map<String, String> additionalLabels = createContainerParams.getContainerConfig().getLabels();
+
+        long memoryLimitBytes = createContainerParams.getContainerConfig().getHostConfig().getMemory();
+        String memoryLimit = Long.toString(memoryLimitBytes / 1048576) + "Mi";
```

Comment:
Ah thank you, I knew I had seen this class before, but couldn't find it very quickly when I was writing this. I'm not sure the output provided by `Size` will work perfectly, since OpenShift interprets `M` as "mega", and `Mi` as mebi. I'll have to test it more.